# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/bash/default-histsize.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- bash-5.2.37/config-top.h.vanilla	2025-06-17 13:57:43.544265393 +0200
+++ bash-5.2.37/config-top.h	2025-06-17 13:59:21.710067593 +0200
@@ -189,7 +189,7 @@
 
 /* Define to set the initial size of the history list ($HISTSIZE). This must
    be a string. */
-/*#define HISTSIZE_DEFAULT "500"*/
+#define HISTSIZE_DEFAULT "10000"
 
 /* Define to 0 if you want history expansion to be disabled by default in
    interactive shells; define to 1 for the historical behavior of enabling
